wordpress-mobile/WordPress-Android
# Issue/6253 adjust cancelled uploads

Fixes #6253

This PR makes sure that we track failed media item uploads and update the post contents accordingly while an upload fail happens in the background. The solution implemented here corresponds to the second option as described in the issue.

To test:

1. create a new draft
2. include media items
3. exit the editor
4. turn airplane mode ON
5. edit the post again: you should see media items marked as FAILED
 
cc @aforcier
